### ISSUE TITLE
Add alt text to all images which are missing it

### ISSUE THREAD

**Issue by mistyhacks** [docker/docker.github.io#4998]
### Problem description

For accessibility, it is good for all images to have alt text.

Do a search for the following string and add the alt text into the empty brackets:

`
![](
`

For instance, an image with missing alt text might look like:

```
![](foo.png)
```

And to fix it, you'd change it to something like:

```
![an example of foo](foo.png)
```

it's best if the alt text is useful to someone who can't actually see the image, but may be using a screen reader or other assistive text. So don't just repeat the image name or something like that.

This is not a difficult task but will take some time due to considering the right alt text to use for each image.

**Comment by bhavin192** [comment]
I will work on this one.

**Comment by bhavin192** [comment]
There are 577 instances of `![](` 
is that okay if I create a WIP PR ?

**Comment by mstanleyjones** [comment]
Absolutely! If it has WIP in the title we won't consider it for merge yet.
